ImageGalery/GaleryTableViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  GaleryTableViewController.swift
+//  ImageGalery
+//
+//  Created by apple on 18/07/2018.
+//  Copyright © 2018 wolfpack. All rights reserved.
+//
+
+import UIKit
+
+class GaleryTableViewController: UITableViewController, UICollectionViewDelegate, galleryUpdateDelegate {
```

Comment:
put 'UICollectionViewDelegate' and 'galleryUpdateDelegate' in a separate 'GaleryTableViewController' class extension for a better code readability.